Failure:

`
> Task :compileIntegrationTestJava
/home/travis/build/apache/fineract/fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/StaffTest.java:135: warning: [unchecked] unchecked cast
        final List<HashMap> responseActive = (List<HashMap>)StaffHelper.getStaffListWithState(requestSpec, responseSpec, "active");
                                                                                             ^
  required: List<HashMap>
  found:    Object
/home/travis/build/apache/fineract/fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/StaffTest.java:144: warning: [unchecked] unchecked cast
        final List<HashMap> responseInactive = (List<HashMap>)StaffHelper.getStaffListWithState(requestSpec, responseSpec, "inactive");
                                                                                               ^
  required: List<HashMap>
  found:    Object
2 warnings
`
See Travis log: https://travis-ci.org/github/apache/fineract/builds/684752992

Seems another Integration Test is Flaky?

--- a/gradle.properties
+++ b/gradle.properties
@@ -21,4 +21,3 @@
 # The daemon is more pain than gain, because the Tomcat in the Gradle process does not shut down cleanly.
 # TODO remove this after https://issues.apache.org/jira/browse/FINERACT-764
 org.gradle.jvmargs=-Xmx4g
-org.gradle.daemon=false

